java/org/apache/tomcat/dbcp/dbcp2/managed/ManagedConnection.java

```diff
@@ -1,4 +1,4 @@
-/**
+vi /**
```

Comment:
I will remove this after your review...

Reply:
No worries. The git repo is only a mirror so I have to manually apply to the patch to svn anyway. I'd already found this one and fixed it.